Generate.py

```diff
@@ -378,13 +378,44 @@ def roll_linked_options(weights: dict) -> dict:
     return weights
 
 
+def compare_results(
+        yaml_value: Union[str, int, bool, dict, list],
+        trigger_value: Union[str, int, bool, dict, list],
+        comparator: str):
+    if yaml_value is None:
+        return False
+    if isinstance(yaml_value, str|bool|int) and isinstance(trigger_value, str|bool|int):
```

Comment:
Union types in `isinstance` are only available starting in Python 3.10
```suggestion
    if isinstance(yaml_value, (str, bool, int)) and isinstance(trigger_value, (str, bool, int)):
```

Reply:
Fixed

Reply:
No, you used a Union, which isn't supported

Reply:
Got it this time. Sorry about that.